public/src/js/models/config/collection.js

```diff
@@ -72,6 +73,23 @@ export default class ConfigCollection extends DropTarget {
         this.typePicker = this._typePicker.bind(this);
     }
 
+    getPlatform() {
+        switch (this.meta.platform()) {
+            case 'App': return 'app only';
+            case 'Web': return 'web only';
+            default: return 'any';
+        }
+    }
+
+    getDisplayName() {
+        const platform = isPlatformSpecificCollection(this.meta.platform()) ? ` (${this.meta.platform()} only)` : '';
+        return this.meta.displayName() + platform || '(no title)';
+    }
+
+    thisIsPlatformSpecificCollection() {
```

Comment:
Why is this a function rather than a constant?

Reply:
I assumed it's necessary to pick up on updated values, but perhaps it's only ever called once on creation anyway?

Reply:
Yes unless you need it to change because a user did something to the collection but it could be an observable in that case.